_pages/contact.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: "Contact Us"
+
+layout: footerimage
+footerimage: "contact.JPG"
```

Comment:
I'm also unsure about the choice of image, so I'm reluctant to merge this since we'd be adding a 1mb image to the codebase (we should optimise images too)